V2/tests/api/azure/time_entry_azure_endpoints_test.py

```diff
@@ -76,3 +76,61 @@ def test__delete_time_entries_azure_endpoint__returns_a_status_code_400__when_ti
 
     assert response.status_code == 400
     assert response.get_body() == b'Invalid Format ID'
+
+
+def test__time_entry_azure_endpoint__returns_all_time_entries(
+     test_db, time_entry_factory, insert_time_entry, activity_factory, insert_activity
+):
+    inserted_activity = insert_activity(activity_factory(), test_db)
+    time_entries_to_insert = time_entry_factory(activity_id=inserted_activity.id, technologies="[jira,sql]")
+    inserted_time_entries = insert_time_entry(time_entries_to_insert, test_db).__dict__
+
+    req = func.HttpRequest(method="GET", body=None, url=TIME_ENTRY_URL)
+
+    response = azure_time_entries.get_time_entries(req)
+    time_entries_json_data = response.get_body().decode("utf-8")
+    time_entry_list = json.loads(time_entries_json_data)
+
+    assert response.status_code == 200
+    assert time_entry_list.pop() == inserted_time_entries
+
+
+def test__time_entry_azure_endpoint__returns_an_time_entry__when_time_entry_matches_its_id(
+     test_db, time_entry_factory, insert_time_entry, activity_factory, insert_activity
+):
+    inserted_activity = insert_activity(activity_factory(), test_db)
+    time_entries_to_insert = time_entry_factory(activity_id=inserted_activity.id, technologies="[jira,sql]")
+    inserted_time_entries = insert_time_entry(time_entries_to_insert, test_db).__dict__
```

Comment:
These three lines could be placed in a setup method to avoid repeating in every test.